newm-server/src/main/kotlin/io/newm/server/aws/s3/S3Ext.kt

```diff
@@ -0,0 +1,120 @@
+package io.newm.server.aws.s3
+
+import com.amazonaws.auth.AWSSessionCredentials
+import com.amazonaws.services.s3.AmazonS3
+import io.ktor.util.*
+import io.newm.server.aws.s3.model.*
```

Comment:
We like to avoid wildcard imports. you could configure it in the IntelliJ preferences.